app/controllers/registrations_controller.rb

```diff
@@ -0,0 +1,12 @@
+class RegistrationsController < Devise::RegistrationsController
+
+  private
+
+  def sign_up_params
+    params.require(:user).permit(:provider_name, :email, :password, :password_confirmation)
+  end
+
+  def account_update_params
+    params.require(:user).permit(:provider_name, :email, :password, :password_confirmation)
```

Comment:
Metrics/LineLength: Line is too long. [91/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)